atlasdb-client/src/main/java/com/palantir/atlasdb/table/description/Schemas.java

```diff
@@ -105,30 +104,19 @@ public static boolean isTableNameValid(String tableName) {
      * (e.g., it is not the responsibility of this method to perform schema
      * upgrades).
      */
-    public static void createTablesAndIndexes(Schema schema, KeyValueService kvs) {
+    public static void createTablesAndIndexes(Schema schema, DdlManager ddlManager) {
         schema.validate();
-        createTables(kvs, schema.getTableDefinitions());
-        createIndices(kvs, schema.getIndexDefinitions());
+        createTables(ddlManager, schema.getTableDefinitions());
+        createIndices(ddlManager, schema.getIndexDefinitions());
     }
 
-    public static void deleteTablesAndIndexes(Schema schema, KeyValueService kvs) {
```

Comment:
Is used internally by large internal product, unfortunately

Reply:
I moved this to TestSchemas which they should have access to surely? Can we just fix there? I don't think this type of code should be in the main atlas API, feels like too blunt of an instrument? Unless you're saying the usages are actually from prod code?

Reply:
Oh wait I just deleted, will move 🤦